perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -340,21 +352,46 @@ def Run(benchmark_spec):
 
   fio_command = 'sudo %s --output-format=json %s' % (fio.FIO_PATH,
                                                      REMOTE_JOB_FILE_PATH)
+
+  run_reps = FLAGS.run_for_minutes // MINUTES_PER_JOB
```

Comment:
This is not always correct. What if there are more than 1 scenarios in generated job file? Each scenario takes 10 minutes.